### PR TITLE
belindas-closet-nextjs_6_323_mui-creator-page

### LINKED ISSUE #323: Creator Page Styling: Integrate MUI on the creator page
Relates to #322 

- [ ] Integrate MUI on the creator page.

### PR DESCRIPTION
**Pull request by nwm516**
Resolves #323 

Integrate Material UI on the Creator page.

https://github.com/SeattleColleges/belindas-closet-nextjs/assets/36825393/80bd082a-8d3d-4ecc-a5bc-bfeed0810e78

Before: 
Admin Page  Placeholder Creator Page

After:
Admin Page MUI Creator Page MUI

### CODE REVIEW

**Comment by taylorpapke** [comment]
Same as your other PR. Don't know if you were going to do this but looks like you just need to do an npm install of the 2 modules shown in the error message below and then push to this PR branch:
Screenshot 2024-05-05 at 2 35 20 PM

**Review by MaddasaHatter — DISMISSED** [review]
Looks like it is getting hung up on one of the checks and failing. I'm also running into the same issue as Taylor when testing the build.

**Review by Asfand00 — DISMISSED** [review]
/This is what the creator page looks on my end: 
creator-page2
For the things circled in red, is that intentional of how it should look, if so it looks kinda hard to see if it's supposed to be purple.

Also when i click on the products dropdown on navbar this shows up:
products
looks like that needs to be fixed

These errors might also be on my end only (not sure), I use npm not yarn, so I had to adjust to those changes and making sure dependecies were right. 

Though it seems your still having the issue with the check not going through because your missing some modules that need to be installed so it seems at the moment that issue is still there.

**Review by Asfand00 — COMMENTED** [review]
Use my earlier comment as general feedback without explicit approval (this is what i meant to do instead) the requested changes were already requested basically.

**Comment by taylorpapke** [comment]
I didn't notice the yarn.lock file that @Asfand00 mentioned. I don't know too much about yarn but after a quick search online, it appears that having 2 package managers (npm and yarn) could potentially lead to dependency conflicts. I'm not sure if that's the case here but we might want to ask ourselves if this is something we want to introduce at this point @brinkbrink

**Comment by brinkbrink** [comment]
> I didn't notice the yarn.lock file that @Asfand00 mentioned. I don't know too much about yarn but after a quick search online, it appears that having 2 package managers (npm and yarn) could potentially lead to dependency conflicts. I'm not sure if that's the case here but we might want to ask ourselves if this is something we want to introduce at this point @brinkbrink

@taylorpapke Yes, multiple package managers can, like you said, lead to conflicts and add unnecessary complexity. That's actually why there's a failing check here. If you click the "Details" link next to the failing check you can see that here:

Screenshot 2024-05-10 at 8 28 28 AM

While it's possible to resolve these dependencies manually, I don't think anybody wants to do that. Especially when you look at the size of that yarn.lock, woof! 

We also had a discussion a while back about not using [dependabot](https://github.com/dependabot) anymore because we wanted to produce stable releases so let's also be careful about changing the package.json unless it's necessary.

Screenshot 2024-05-10 at 8 30 58 AM

This is stuff that not everybody has been exposed to/learned about so I'm kind of glad you did this @nwm516 so we can all use this as a learning experience.

**Review by brinkbrink — CHANGES_REQUESTED** [review]
There are definitely benefits to using yarn and if you think this is a direction the team should go in we should have a team discussion but for now we are using npm as our package manager. Please make appropriate changes. Feel free to reach out to @tinpham5614, @IsaacJrTypes or @cshimm for help with these changes if you hit a roadblock. Hopefully it isn't too painful.

**Comment by nwm516** [comment]
Thanks everyone for the feedback. I put this on the backburner with the intent of returning once I had found the courage to try again, heh. 

Makes sense on dependencies from yarn and npm conflicting, though, like you said @brinkbrink, this is something that is wholly new to me, so my understanding of how this all plays together is still very fresh.

I'll make the appropriate changes again with integrating npm solely and will reach out to the appropriate team members with any difficulties I encounter here on out.

**Comment by brinkbrink** [comment]
> Thanks everyone for the feedback. I put this on the backburner with the intent of returning once I had found the courage to try again, heh.
> 
> Makes sense on dependencies from yarn and npm conflicting, though, like you said @brinkbrink, this is something that is wholly new to me, so my understanding of how this all plays together is still very fresh.
> 
> I'll make the appropriate changes again with integrating npm solely and will reach out to the appropriate team members with any difficulties I encounter here on out.

Sounds good! Walking away and coming back is sometimes the best solution, I've been there...quite a few times haha

**Review by taylorpapke — APPROVED** [review]

**Review by Asfand00 — APPROVED** [review]
LGTM! everything checks out on my end 
creator-page3

**Review by el-riber — APPROVED** [review]
you code looks great!

**Review by intisarosman1 — APPROVED** [review]
Everything looks good!
Screenshot 2024-05-16 214917

File: app/creator-page/page.tsx
Comment (gitbiruk2010) [inline app/creator-page/page.tsx:1]: The updates to the creator-page/page.tsx are looking good! The use of Material-UI components like Button, Typography, and ButtonGroup greatly enhances the UI, making it more user-friendly and visually appealing. The refactor to remove boilerplate code and streamline the component structure is a good improvement. Additionally, the well-organized addition of placeholders and clear navigation buttons simplifies the user journey. Great job!

**Review by heosman — APPROVED** [review]
It looks good now and the buttons are routing to the correct pages.

**Review by nickolasram — APPROVED** [review]
Great job sorting everything out! Everything looks great on my end

**Comment by taylorpapke** [comment]
@brinkbrink Can you approve changes? It looks like 3 approvals have been met for this PR.

**Review by brinkbrink — APPROVED** [review]

**Pull request merged**